packages/styletron-react/src/index.js

```diff
@@ -85,6 +86,43 @@ class DevProvider extends React.Component<
 export const Provider =
   __BROWSER__ && __DEV__ ? DevProvider : StyletronContext.Provider;
 
+// DEVTOOLS SETUP
+type StyletronStyles = {
+  classes?: any,
+  styles?: any,
+  extends?: any,
+};
+if (__BROWSER__ && __DEV__ && !window.__STYLETRON_DEVTOOLS__) {
+  const atomicMap = new Object();
```

Comment:
I think this is also worth moving into a separate function `setupDevtools()` and moving into the seame